### PR TITLE
Docs: Add minimal runnable example for GraphConvModel tutorial (#4500)

### LINKED ISSUE #4500: [Documentation] Missing example in GraphConv tutorial
### Problem
The `GraphConvModel` section in tutorials (http://deepchem.io/docs) explains theory but lacks a minimal runnable example. New users may struggle to instantiate the model or load data.

### Steps to Reproduce
1. Visit the documentation section for `GraphConvModel`  
2. Notice there is no code snippet demonstrating dataset loading, model training, or evaluation

### Expected Behavior
Include a code snippet like:
```python
from deepchem.models import GraphConvModel
from deepchem.data import NumpyDataset

# Example data
X, y = load_some_graph_data()
model = GraphConvModel(...)
model.fit(dataset)
print(model.evaluate(dataset, ["accuracy"]))

### PR DESCRIPTION
**Pull request by dakshhhhh16**
## Description

Fixes #4500  

This PR adds a minimal runnable example to the **GraphConvModel** tutorial.  
The current documentation explains theory but lacks an executable snippet showing how to instantiate, train, and evaluate a `GraphConvModel`.  
This example helps new users quickly understand the workflow and run the model without setup confusion.

The snippet demonstrates:
- Creating a small dummy dataset using `NumpyDataset`
- Initializing `GraphConvModel` for binary classification
- Fitting the model on sample data
- Evaluating the model with a simple metric (`accuracy`)

---

## Type of change

Please check the option that is related to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentations (modification for documents)

---

## Checklist

- [x] My code follows [the style guidelines of this project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have checked my code and corrected any misspellings
- [x] The snippet follows the same structure and formatting conventions used in other DeepChem tutorials

### CODE REVIEW

**Comment by sudo-rsingh** [comment]
@dakshhhhh16 Thanks for the PR, but if the example is in doc string then we don't need an example here.
